debian/control

```diff
@@ -0,0 +1,13 @@
+Package: cometvisu
+Replaces: cometvisu-wiregate,cometvisu-0.5,cometvisu-0.8
+Priority: extra
+Maintainer: CometVisu.org <info@cometvisu.org>
+Vcs-Git: https://github.com/CometVisu/CometVisu.git
+Vcs-browser: https://github.com/CometVisu/CometVisu
+Homepage: http://www.cometvisu.org/wiki/CometVisu
+Architecture: all
+Section: net
+Depends: libapache2-mod-php5 | libapache-mod-php5 | php5-cgi | php5, rrdtool (>= 1.3.1-4+nmu1), cometvisu-backend | knxd-tools
```

Comment:
php is only recomended, not required. (Only plugins can depend on it, all basic features should work on a simple system w/o PHP installed, like a little router)